common/src/main/java/org/dash/wallet/common/Configuration.java

```diff
@@ -553,14 +553,6 @@ public String getLastCoinbaseRefreshToken() {
         return prefs.getString(PREFS_KEY_LAST_COINBASE_REFRESH_TOKEN, null);
     }
 
-    public void setLastCoinbaseBalance(String balance) {
-        prefs.edit().putString(PREFS_KEY_LAST_COINBASE_BALANCE, balance).apply();
-    }
-
-    public String getLastCoinbaseBalance() {
-        return prefs.getString(PREFS_KEY_LAST_COINBASE_BALANCE, null);
-    }
-
```

Comment:
I figured we should start moving feature-specific prefs from here into dedicated configs. The last balance seems like a good one to start with since it's not critical if something goes wrong with it.